style.css

```
@@ -0,0 +1,334 @@
+}
+
+.header_container {
+  z-index: 10;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1920px;
+  width: 100%;
+}
+
+.header_logo {
+  cursor: pointer;
+}
+
+.header_login {
+  width: 128px;
+  padding: 16px 20px;
+  border-radius: 8px;
+  background: linear-gradient(91deg, #6D6AFE 0.12%, #6AE3FE 101.84%);
+  
+  color: #F5F5F5;
+  font-family: Pretendard;
+  font-size: 18px;
+  font-weight: 600;
+  text-align: center;
+
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.lineBreak_tablet {
+  display: none;
+}
+
+.banner {
+  display: flex;
+  justify-content: center;
+  background-color: #F0F6FF;
+}
+
+.banner_container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 40px;
+  
+  width: 100%;
+  padding-top: 70px;
+  overflow: hidden;
+}
+
+.banner_text {
+  margin: 0;
+  text-align: center;
+  font-family: Pretendard;
+  font-size: 64px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 80px
+}
+
+.banner_gradient {
+  background: linear-gradient(91deg, #6D6AFE 17.28%, #FF9F9F 74.98%);
+  color:transparent;
+  background-clip: text; 
+}
+
+.button_addlink {
+  background: linear-gradient(91deg, #6D6AFE 0.12%, #6AE3FE 101.84%);
+
+  width: 350px;
+  padding: 16px 20px;
+  border-radius: 8px;
+  
+  color: #F5F5F5;
+  font-family: Pretendard;
+  font-size: 18px;
+  font-weight: 600;
+  text-align: center;
+  text-decoration: none;
+}
+
+.content.first {
+  padding-top: 120px;
+}
+
+.content {
+  display: flex;
+  justify-content: center;
+  padding: 50px 0;
+}
+
+.content_container {
+  display: flex;
+  align-items: center;
+  gap: 157px;
+  justify-content: space-between;
+}
+
+.content_main {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 10px;
+  width: 315px;
+}
+
+.content_title {
+  margin: 0;
+  font-family: Pretendard;
+  font-size: 48px;
+  font-weight: 700;
+}
+
+.content_1_gradient {
+  background: linear-gradient(96deg, #FE8A8A 1.72%, #A4CEFF 74.97%);
+  color:transparent;
+  background-clip: text;
+}
+
+.content_text {
+  margin: 0;
+  color: #6B6B6B;
+  font-family: 'Abel', sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 150%;
+}
+
+.content_image {
+  width: 550px;
+  height: 450px;
+}
+
+.content_2_gradient {
+  background: linear-gradient(277deg, #6FBAFF 59.22%, #FFD88B 93.66%);
+  color:transparent;
+  background-clip: text;
+}
+
+.content_3_gradient {
+  background: linear-gradient(95.63deg,#6d7ccd 1.72%,#528885 74.97%);
+  color:transparent;
+  background-clip: text;
+}
+
+.content_4_gradient {
+  background: linear-gradient(95.63deg,#fe578f 1.72%,#68e8f9 74.97%);
+  color:transparent;
+  background-clip: text;
+}
+
+footer {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  background-color: #111322;
+  margin-top: 120px;
+  padding: 32px 104px 64px;
+  height: 160px;
+}
+
+.codeit {
```

부모 요소까지 포함해 스타일이 중복되는 사이드 이펙트를 사전에 없애는것도 좋을것 같아요!
```suggestion
footer .codeit {
```